src/components/Components/LinkedSecretsListView/LinkedSecretsListHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import { createTableHeaders } from '../../../shared/components/table/utils';
+
+export const linkedSecretsTableColumnClasses = {
+  secretName: 'pf-m-width-35 wrap-column',
+  type: 'pf-m-width-30 wrap-column',
+  labels: 'pf-m-width-30 wrap-column',
+  actions: 'pf-m-width-30 wrap-column',
```

Comment:
I checked the secret list table, it just enjoys wrap-column for name & labels.
i guess for fixed short columns we do not need to consider wrap, maybe?

Reply:
good catch, updated!